integration/inspektor-gadget/integration_test.go

```diff
@@ -382,10 +383,29 @@ func TestBiotop(t *testing.T) {
 	t.Parallel()
 
 	biotopCmd := &Command{
-		Name:           "StartBiotopGadget",
-		Cmd:            fmt.Sprintf("$KUBECTL_GADGET top block-io -n %s", ns),
-		ExpectedRegexp: `test-pod\s+test-pod\s+\d+\s+dd`,
-		StartAndStop:   true,
+		Name:         "StartBiotopGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET top block-io -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &biotopTypes.Stats{
+				CommonData: BuildCommonData(ns),
+				Write:      true,
+				Comm:       "dd",
+				Bytes:      2097152,
+				Operations: 2,
```

Comment:
> Is the Operations always guaranteed to be 2?
> 
I don't think so. @eiffel-fl any comment?

Reply:
Thanks for all the information :)
I'll keep the deviations from the regex to a minimum in future PRs